docs/gettsim_objects/input_variables.md

```diff
@@ -486,3 +462,37 @@ Type: float
 Received Bürgergeld in previous year
 
 Type: bool
+
+(anwartschaftszeit)=
+
+## `anwartschaftszeit`
```

Comment:
```suggestion
## `arbeitsl_geld_anwartschaftszeit`
```

Apologies I missed that one. `anwartschaftszeit` is probably more common in pensions context, so we'll definitely need a prefix. Probably need to make it private or ignore errors for the moment.

Reply:
Sure, we just need to be explicit here.

Reply:
Ok 👍

Reply:
> `anwartschaftszeit` is probably more common in pensions context, so we'll definitely need a prefix. Probably need to ignore errors for the moment.

I checked again and `anwartschaftszeit` is indeed only used for Arbeitslosengeld (pensions use the term Anwartschaft though).

Should we really increase `GEP_01_CHARACTER_LIMIT_USER_FACING_COLUMNS` to 32 (like for all other columns) for this exception? Would be fine with me, but I would prefer going with `anwartschaftszeit` only.

Reply:
Thanks! Just go for it as is, distinction will be irrelevant in the near future, anyhow.

Reply:
And apologies, @paulinaschroeder, for not reading your earlier message carefully. I actually had not realised the difference!